ONSdigital/eq-survey-runner
# Eq 410 navigator

### What is the context of this PR?

Removing the navigator and replacing it with the User Journey Manager. Also cleaning up the User action processor and moving the routing components into the routing engine. 
### How to review
1. Check out this branch
2. Check the application still works
3. Ensure all tests pass
### Who worked on the PR

@warrenbailey
